src/main/java/com/pinterest/secor/common/SecorConfig.java

```diff
@@ -363,6 +363,10 @@ public String getUploadManagerClass() {
         return getString("secor.upload.manager.class");
     }
 
+    public String getInnerUploadManagerClass() {
+        return getString("secor.inner.upload.manager.class");
```

Comment:
The code looks fine, just be sure your keys are right cause I don't know which ones you need here.

Reply:
👍